includes/user.php

```diff
@@ -22,8 +24,14 @@ class User {
 	 */
 	const ADMIN_NOTICES_KEY = 'elementor_admin_notices';
 
+	/**
+	 * @deprcated use User_Introduction
```

Comment:
Also in other places.

```suggestion
	 * @deprecated 3.5.0 use `User_Introduction::get_key()` instead.
```